specs/notation-cli.md

```diff
@@ -5,8 +5,7 @@ This spec contains reference information on using notation commands. Each comman
 ## Notation Commands
 
 | Command                                     | Description                                                            |
-| ------------------------------------------- | ---------------------------------------------------------------------- |
-| [blob](./commandline/blob.md)               | Sign, verify and inspect singatures associated with blobs              |
```

Comment:
This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

Reply:
Tracking: https://github.com/notaryproject/notation/issues/1012